src/components/layout/NotificationSnackBar/DonationNotificationLayout.tsx

```diff
@@ -0,0 +1,38 @@
+import { useTranslation } from 'next-i18next'
+import { Avatar, Box, Typography } from '@mui/material'
+import { dateToTime } from 'components/faq/helpers/utils'
+import { grey } from '@mui/material/colors'
+import { money } from 'common/util/money'
+
+export type NotificationLayoutData = {
+  createdAt: string
+  person: {
+    firstName: string
+    lastName: string
+    picture: string
+  }
+  currency: string
+  amount: number
+}
+
+export default function DonationNotificationLayout({
+  data,
+}: {
+  data: NotificationLayoutData | undefined
+}) {
+  const { person, createdAt, currency, amount } = data || {}
+
+  return (
+    <Box sx={{ display: 'flex', alignItems: 'center', maxHeight: '50px' }}>
+      <Avatar sx={{ marginRight: '1rem' }} src={person?.picture} alt="Avatar" />
```

Comment:
`person?.picture` here will not be avaliable a lot of the time. You need some kind of fallback. Either the first letters of their names or a generic avatar picture

Reply:
The screenshot I provided shows it when we have a missing person?.picture

Reply:
Yeah didn't notice that. 👍